eng/Directory.Build.Data.targets

```diff
@@ -10,15 +10,19 @@
     <Error Condition="'@(MissingTargetFrameworks)' != ''" Text="Missing required target frameworks '@(MissingTargetFrameworks)'. Please ensure you add those frameworks." />
   </Target>
 
-  <!-- Set PackageProjectUrl to the package README for Client Libraries -->
-  <Target Name="SetPackageProjectUrl" BeforeTargets="ValidateTargetFrameworks" Condition="'$(IsClientLibrary)' == 'true' and '$(IsTestProject)' != 'true' and
+  <!-- Set PackageProjectUrl and PackageReleaseNotes to the package README.md and CHANGELOG.md respectively for DataPlane Libraries -->
+  <Target Name="SetPackageProjectUrlandReleaseNotes" BeforeTargets="ValidateTargetFrameworks" Condition="'$(IsDataPlaneProject)' == 'true' and '$(IsTestProject)' != 'true' and
    '$(IsSamplesProject)' != 'true' and '$(IsTestSupportProject)' != 'true' and '$(CommitSHA)' != ''">
     <PropertyGroup>
       <CurrentCsprojPath>$(MSBuildProjectDirectory)</CurrentCsprojPath>
       <DirectoryPartofPath>$(CurrentCsprojPath.Substring($(CurrentCsprojPath.LastIndexOf("\sdk\"))))</DirectoryPartofPath>
       <DirectoryPartofPath>$(DirectoryPartofPath.Remove($(DirectoryPartofPath.LastIndexOf("\src"))))</DirectoryPartofPath>
       <PackageProjectUrl>$(RepositoryUrl)/blob/$(CommitSHA)$(DirectoryPartofPath)/README.md</PackageProjectUrl>
+      <PackageReleaseNotes Condition="'$(PackageReleaseNotes)' == '' and 'Exists('$(RepoRoot)/$(DirectoryPartofPath)/CHANGELOG.md')''">$(RepositoryUrl)/blob/$(CommitSHA)$(DirectoryPartofPath)/CHANGELOG.md<PackageReleaseNotes>
```

Comment:
Didn't you also plan to try and pull out the specific version release notes?

Reply:
That will happen in on of the release steps once the changelog is present in the nuget.

Reply:
How are you going to pull out the changelog snippet and put it into the package metadata during the release step? I know we will do that for the github release but I'd expect for the package we would have to do it here somewhere.

Reply:
Filled Issue to get that done here https://github.com/Azure/azure-sdk-for-net/issues/9179